master/static/srv/shell-entrypoint.sh

```diff
@@ -36,6 +36,8 @@ options="$(
         echo -n "environment=\"$var=$val\","
     done
 )"
-sed -i -e "s/^/$options /" "/run/determined/ssh/authorized_keys"
+unmodified="/run/determined/ssh/authorized_keys_unmodified"
```

Comment:
non-blocking: could be helpful to add in a comment about why you need this

Reply:
done